lib/twilio/call.ts

```diff
@@ -1550,6 +1560,11 @@ namespace Call {
      */
     isUnifiedPlanDefault: boolean;
 
+    /**
+     * A function to be called after {@link Call.ignore} is called.
+     */
+    onIgnore: () => any;
```

Comment:
The other type is `() => void` (which I believe is preferred over `any`). Should we update this one to match?